ScanUnsupportedChars/ScanUnsupportedChars.ps1

```diff
@@ -726,6 +744,30 @@ if ($FilesWithInvalidCharsFixedName.Count -gt 0)
 
         Write-Host "***************************Items failed to rename table end*****************************" -ForegroundColor Yellow
     }
+    elseif($PSBoundParameters.ContainsKey('DestinationPath'))
```

Comment:
Validate the params that renameitem and destinationpath + robocopy options are not specified together. Similarly csvpath is also not needed in case of copy.

Reply:
I've added the check for the renameitems parameter, robocopyoptions is optional to provide further settings, see the readme for more information about it.

Reply:
Regarding the CSVPath Parameter, I've wanted to leave this as potential option to generate a report of which files had been identified for later investigation. The new code should be non-invasive to leave this functionallity working - only with the re-naming the copy would not work as files are identified by original name and not the fixed version